static/.htaccess

```diff
@@ -1,29 +1 @@
-ExpiresActive on
 FileETag MTime
```

Comment:
Whitenoise doesn't provide Etag support out of the box since it's by design something that is left to the server to implement. The format of Etag heavily differ between server software, given our jump away from Apache soon as part of the AWS move (going to Nginx) we'll need to reevaluate the use of Etag's anyway. I'd say we remove them and only live with the ones that Whitenoise sets: `Last-Modified`, `Content-Length`, `Content-Type`, `Content-Encoding`, `Cache-Control`. That should give our CDN enough info to decide whether to fetch the files or not.